### DISCUSSION TITLE
🎉 Welcome to the new home for the date-fns community!

### DISCUSSION THREAD

**Discussion by kossnocorp** [js-fns/date-fns#1798] [General]
Hey everyone,

Please welcome to the new home for the date-fns community!

Previously, we had [a community at Spectrum](https://spectrum.chat/date-fns), but it didn't manage to take off and became yet another place where __only me__ is answering the questions. On top of that, GitHub acquired Spectrum, and the project became abandoned.

I hope that GitHub Discussions will be used as the go-to place to get and provide help with date-fns and JavaScript dates in general and will allow us to keep Issues clean and on point.

If you have something that concerns you, have an interesting topic to discuss (given it's somehow relevant to date-fns), or have an idea, I encourage you to start a discussion.

Cheers,
Sasha

**Comment by unr** [comment]
Worth noting the official date-fns page still links to Spectrum in the header. 👍 

https://spectrum.chat/date-fns

**Reply by kossnocorp** [reply]
That's a good point! I forgot to update it.

**Reply by scscgit** [reply]
It still links to Spectrum :)

**Reply by ralyodio** [reply]
hopefully this gets more usage.
